setup.py

```diff
@@ -9,7 +9,7 @@
     readme = f.read()
 
 version = (
-    [l for l in open(join(PROJECT_ROOT, 'zeroconf', '__init__.py')) if '__version__' in l][0]
+    [ln for ln in open(join(PROJECT_ROOT, 'zeroconf', '__init__.py')) if '__version__' in ln][0]
```

Comment:
This seems out of scope of this pull request.

Reply:
The CI was failing until I fixed this since `l` is apparently no longer an allowed var name.

Reply:
Opened #252 which should allow this to merge cleanly on top if I did it right.